e2e/cli_scan_test.go

```diff
@@ -80,6 +86,23 @@ func TestCLIScan(t *testing.T) {
 			scanImage(t, TestImageName, appID)
 			validateScanImage(t)
 
+			return ctx
+		}).
+		Assess("cli scan flow - image with known bad metadata in cyclonedx", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
```

Comment:
The problem here is that the database already has results from previous test - "cli scan flow - analyze and scan"
You need to either clean up the database, or do a more precise validation (like component id or somthing)

Reply:
I'm using a different application name and ID I thought that would be enough to separate the validation, I need to re-read the validate functions

Reply:
We're now filtering all the validate steps using the appID unique to this test so I think we should be good.